README.rst

```diff
@@ -0,0 +1,34 @@
+Yola API Python Client
+======================
+
+Usage
+-----
+
+.. code:: python
+
+    from yolapy.services import Yola
+
+
+    yola = Yola(
+        url='https://wl.qa.yola.net/',
+        auth=('username', 'password'))
+
+    yola.create_user(
+        email='test@example.com',
+        name='Jane',
+        surname='Doe',
+        partner_id='WL_YOLA',
+        preferences={'name='value'})
+
+See <http://yolapy.readthedocs.org/> for available methods with
```

Comment:
Fixed leftover markdown link syntax in https://github.com/yola/yolapy/commit/61be61cb9840c219894151620cd012318fae0940